src/app.js

```
@@ -1 +1,83 @@
+/* eslint-disable no-console */
+
 // write code here
+const fs = require('fs');
+
+const path = require('path');
+
+function moveFile(src, dst) {
+  let dstFixed = dst;
+
+  const srcFileName = path.basename(src);
+  const dstFolder = path.dirname(dst);
+
+  if (src === undefined || dst === undefined) {
+    console.error('error if only one argument is provided');
+
+    return;
```

The task description says:
> In case destination directory does not exist an error must be thrown.

However, your code does not handle the case when only one argument is provided and just returns from the function. You need to throw an error to meet the task requirements.